spec/lib/webdriver_script_adapter/exec_eval_script_adapter_spec.rb

```diff
@@ -44,18 +44,6 @@ module WebDriverScriptAdapter
         end
       end
 
-      context "Selenium (doesn't respond to #evaluate_script)" do
```

Comment:
There's probably a lot of wrapper and adapter code that could be cleaned up if watir/selenium/phantom are no longer supported. Not necessary for this, but opportunity for future cleanup.